config/deploy-queue-proxy.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2019 The Knative Authors
```

Comment:
Need to double-check that this doesn't overwrite the existing config-deployment... might be better to just have the setup-guard-service script patch the existing configmap with the current QP image...

Reply:
I tested by adding a `test: boom` parameter to config.deployment  - this parameter remains when doing security-guard `ko apply -Rf ./config` while the image changes  - note that the _example is removed after such a change. 

after `ko apply -Rf config/core` at serving 
```
data:
  _example: |-
    ################################
    #                              #
    #    EXAMPLE CONFIGURATION     #
    #                              #
    ################################

    # This block is not actually functional configuration,
    ...
    # NOTE THAT THIS IS AN EXPERIMENTAL / ALPHA FEATURE
    concurrency-state-endpoint: ""
  queueSidecarImage: us.icr.io/knat/queue-39be6f1d08a095bd076a71d288d295b6@sha256:086b8a7d8bc2104672609a1918274f2e4b075076f52c0fbbc4c3f044b64eb5fe
  test: boom
```
after `ko apply -Rf ./config`  at security-guard
```
data:
  queue-sidecar-image: us.icr.io/knat/queue-958f2fa0da7e00e1e412a537ecf9d1cc@sha256:021e6438111346ea5d5aff064d6c3ad2cea6c3a92cc7e7708f644f8a1fc34a1e
  test: boom
```